src/besseli.jl

```diff
@@ -130,6 +130,238 @@ function besseli1x(x::T) where T <: Union{Float32, Float64}
+                  -0.3541218766747901 + 5.402855776372861im, -0.9973883749616251 + 15.217205990064667im, 
+                  5.067285183070908 + 3.256546447342955im, 2.1309472708925585 + 19.986722185708082im, 
+                  -0.8162780120038937 + 12.453995821138065im, -0.6642234573493526 + 10.134091621337502im
+                  )
+            w  = (0.023435912479136792 + 0.0im, -0.07518141047674767 + 0.0475948197731169im, 
+                  0.02099862365431344 + 0.10227799317706024im, -0.044790606134137025 - 0.0019900581421525287im, 
+                  0.05092855107420865 - 0.030284027691088355im, 0.11876822116350058 + 0.3545737241031229im, 
+                  -0.2800428715703842 + 0.1385502878368521im, 0.07056156978707125 - 0.015640049217372977im, 
+                  -0.06946091028316168 + 0.016066454632345423im, 0.08524646892263366 + 0.0953622756709394im, 
+                  -0.02790219158045262 + 0.07064926802304639im, 0.1422314233425249 - 0.2536760604980481im, 
+                  -0.20586363442913913 + 0.3014383126557577im, 0.20547145538054265 - 0.17388658146749558im, 
+                  -0.13839690210608482 - 0.07997812132261209im, -0.005116954351018287 + 0.07859682420061695im, 
+                  0.0409490985507271 - 0.2642881024594653im, -0.04174610452138103 - 0.23810783641445965im, 
+                  -0.14078210159561536 - 0.17647153499232213im, -0.07672294420555978 + 0.08789101702567167im, 
+                  -0.15544491331665283 - 0.17872134246439533im, 0.2746212867658749 + 0.0633913060138901im, 
+                  0.003561860807958012 + 0.005781000943183686im, 0.22467614820711904 + 0.050873741433048215im
+                  )
+            f = (4.11757504005866 - 4.095512734321109im, -2.247112620034342 + 3.680759784557969im, 
+                 3.7834318298173315 + 0.562243565983836im, 1.025044899237516 - 2.4295763517152302im, 
+                 -1.1024728966418114 - 0.9985456859791928im, -0.7134052191604513 + 0.634168698676205im, 
+                 0.35489432819051814 + 0.8984965319188546im, 0.15105140195420463 - 0.7254114647779558im, 
+                 0.401057852147248 - 0.0014084695668244449im, 0.5404571564588408 - 0.1885021348797114im,
+                 0.407770977742567 - 0.00680290179701475im, 0.3994965367795384 - 0.0024661141779376626im, 
+                 0.8312271064319111 - 1.0050503936637845im, 0.40006191058465196 - 0.002248719754950748im, 
+                 0.4055260568863696 - 0.01689191195791546im, 5.210832430161148 + 1.7413278357835171im, 
+                 0.40364747476765517 - 0.003288380266379im, 0.40062798907705705 - 0.0018845654535970921im, 
+                 -0.39383201186515915 - 0.17987628384333437im, -2.053184226968296 + 1.6069066509951657im, 
+                 0.40617427977861376 - 0.005320233415147144im, 0.4034434765080089 - 0.006086635360116373im,
+                 -0.07645988937623527 + 1.981923553555365im, 1.8844134122620073 + 0.24179792442042025im
+                ).*w
```

Comment:
Would you get better accuracy by moving the `.*w` to compile time where you could do it in arbitrary precision?